ucm2/sof-soundwire/rt715.conf

```diff
@@ -4,17 +4,17 @@ SectionDevice."Mic" {
 	Comment	"SoundWire microphones"
 
 	EnableSequence [
-		cset "name='PGA5.0 5 Master Capture Switch' 1"
+		cset "name='rt715 Main Capture Switch' 1"
```

Comment:
As @perexg said, the patch is still in discussion. And if ALSA finally accepts the kernel patch, I suggested using 'If'. This can help not block the previous projects.

Reply:
agreed!
I will update this PR with condition judgment to avoid impacting old platforms.

Reply:
> As @perexg said, the patch is still in discussion. And if ALSA finally accepts the kernel patch, I suggested using 'If'. This can help not block the previous projects.

Hi @libinyang 
I update PR with if judgment added.
It is not tested yet and need to be optimized.
Please help to take a look if the judgement framework is good to go.